src/js/hooks/cycleCount/useAllProductsTab.jsx

```diff
@@ -81,6 +83,14 @@ const useAllProductsTab = ({
   const getParams = ({
     sortingParams,
   }) => _.omitBy({
+    statuses: [
+      CycleCountCandidateStatus.CREATED,
+      CycleCountCandidateStatus.REQUESTED,
+      CycleCountCandidateStatus.COUNTING,
+      CycleCountCandidateStatus.COUNTED,
+      CycleCountCandidateStatus.INVESTIGATING,
+    ],
+    showCountAndResolveProducts: true,
```

Comment:
btw I think something like `showCycleCountsInProgress` would sound better than "countAndResolve", since we might have also the approve step in the future and this would have to be changed if the cycle counts at the approval step should also be a part of that statement.